src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -0,0 +1,20 @@
+package lotto.domain
+
+class WinningLotto(val numbers: LottoNumbers, val bonusNumber: Int) {
+    val size: Int
+        get() = numbers.size
+
+    fun rank(lotto: Lotto): Rank {
+        val countOfMatch = countMatch(lotto)
+        val matchBonus = matchBonus(lotto)
```

Comment:
한 줄로 끝나는 코드들이니 이 부분도 굳이 메서드로 분리할 필요는 없을 것 같아요. 😃
```suggestion
        val countOfMatch = numbers.countMatch(lotto.numbers)
        val matchBonus = lotto.numbers.match(bonusNumber)
```

Reply:
함수를 제거하도록 하겠습니다 :)